src/System.Private.ServiceModel/tools/scripts/StopWcfSelfHostedSvc.cmd

```diff
@@ -25,10 +38,38 @@ if '%_runelevated%' == '' (
 )
```

Comment:
Remark only: Wow, this is confusing as I'm seeing StopWcfSelfHostedProcess.cmd vs StopWcfSelfHostedSvc,cmd; both very similarly named. Any way for us to merge that command here?